src/shared/services/api/api.test.js

```diff
@@ -17,17 +16,46 @@ describe('api service', () => {
   const queryString = 'name1=value1&name2=value2&value3=foo&value3=bar';
   const url = 'https://url/to/test';
   const token = 'bearer-token';
+  let origSessionStorage;
```

Comment:
why do you not use `getAccessToken.mockImplementation(() => 'mock')` here?

Reply:
I've added a package that mocks the sessionstorage instead.

Reply:
die package hoef je niet te gebruiken, als je:

import { getAccessToken } from 'shared/services/auth/auth';
jest.mock('shared/services/auth/auth');

en dan in elke test: 
getAccessToken.mockImplementation(() => true) // of false

dan laad hij geen andere services in.

dat zit ook op andere plekken die van getAccessToken gebruikmaken.

Reply:
Problem is that, when I start mocking that module, I need to make more changes in other, unrelated, components. Even if I restore mocks after the test suite has run. Using the localstorage mock package overcomes that problem.

Reply:
ok laat maar dan